### PR TITLE
MAISTRA-2206 Remove member-of label from SMCP namespace on deletion

### PR DESCRIPTION
**Pull request by luksa**

### CODE REVIEW

File: pkg/controller/common/metadata.go [hunk 1]
```diff
@@ -21,6 +21,9 @@ const (
 	// MemberOfKey represents the mesh (namespace) to which the resource relates
 	MemberOfKey = MetadataNamespace + "/member-of"
 
+	// MemberOfKey represents the mesh (namespace) to which the resource relates
```
Comment (brian-avery) [inline pkg/controller/common/metadata.go]: Comment is wrong here

Comment (luksa) [reply pkg/controller/common/metadata.go]: Thanks! Fixed.

**Review by jwendell — APPROVED** [review]

**Review by brian-avery — APPROVED** [review]

**Comment by dgn** [comment]
Does this also affect 1.1 operators?

**Comment by luksa** [comment]
Most likely yes. I've marked the PR for cherry-picking to 1.1, too. Thanks!

**Pull request merged**